Product/Production/Services/DocumentQueryCore/src/test/java/gov/hhs/fha/nhinc/docquery/audit/transform/DocQueryAuditTransformsTest.java

```diff
@@ -258,4 +280,22 @@ private AdhocQueryRequest createAdhocQueryRequest() {
     protected AuditTransforms<AdhocQueryRequest, AdhocQueryResponse> getAuditTransforms() {
         return new DocQueryAuditTransforms();
     }
+
+    private NhinTargetSystemType createNhinTarget(String hcid) {
+        NhinTargetSystemType targetSystem = new NhinTargetSystemType();
+        targetSystem.setHomeCommunity(createTargetHomeCommunityType(hcid));
+        return targetSystem;
+    }
+
+    private HomeCommunityType createTargetHomeCommunityType(String hcid) {
+        HomeCommunityType homeCommunityType = new HomeCommunityType();
+        homeCommunityType.setHomeCommunityId(hcid);
+        homeCommunityType.setName("SSA");
+        homeCommunityType.setDescription("This is DOD Gateway");
+        return homeCommunityType;
+    }
+
+    private String decodeBase64Value(byte[] encodedBytes) {
+        return new String(encodedBytes);
```

Comment:
Two things:

1)  This doesn't actually decode anything, does it?
2)  Even if it does, do we need a separate function to call "new String(...)"?